cmd/config/olm/olm.yml

```diff
@@ -65,3 +66,68 @@ jobs:
         inputVars:
           NAME_PREFIX: "benchmark-cc"
           catalogImage: {{.CATALOG_IMAGE}}
+
+  - name: create-clusterextensions
+    namespace: {{.NAMESPACE}}
+    jobIterations: {{.JOB_ITERATIONS}}
+    maxWaitTimeout: 2h
+    qps: {{.QPS}}
+    burst: {{.BURST}}
+    namespacedIterations: {{.NAMESPACED_ITERATIONS}}
+    iterationsPerNamespace: {{.ITERATIONS_PER_NAMESPACE}}
+    churn: {{.CHURN}}
+    churnCycles: {{.CHURN_CYCLES}}
+    churnDuration: {{.CHURN_DURATION}}
+    churnPercent: {{.CHURN_PERCENT}}
+    churnDelay: {{.CHURN_DELAY}}
+    churnDeletionStrategy: {{.CHURN_DELETION_STRATEGY}}
+    executionMode: parallel
+    verifyObjects: true
+    errorOnVerify: true
+    skipIndexing: false
+    podWait: true
+    waitWhenFinished: true
+    preLoadImages: true
+    preLoadPeriod: 2m
+    namespaceLabels:
+      security.openshift.io/scc.podSecurityLabelSync: false
+      pod-security.kubernetes.io/enforce: privileged
+      pod-security.kubernetes.io/audit: privileged
+      pod-security.kubernetes.io/warn: privileged
+    objects:
+
+      - objectTemplate: catalogd.yml
+        replicas: 1
+        inputVars:
+          prefixImageName: {{.PREFIX_IMG_NAME}}
+        waitOptions:
+          customStatusPaths:
+            - key: ".conditions[].type"
```

Comment:
This is a loose match, which leads to false positives. Please use the specific one:
```console
            - key: ".conditions[?(@.type=='Serving')].status"
              value: "True"
```

Reply:
The same as https://github.com/kube-burner/kube-burner-ocp/pull/282/files#r2299960106